### PR TITLE
Removes /tg/

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

The cargo computer used to say "/tg/ station 13 Cargo Department". Now it says "[station-name-here] Cargo Department"

image

🆑 
rscadd: The stock market console now actually displays where you are!!!
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
good branch name

File: code/game/objects/items/devices/PDA/cart.dm [hunk 1]
```diff
@@ -661,7 +661,7 @@
 			menu += "</ol><font size=\"-3\">Upgrade NOW to Space Parts & Space Vendors PLUS for full remote order control and inventory management."
 		if (56) //quartermaster stock viewer
 			menu = "<h4><img src=pda_crate.png> Stock Exchange Viewer</h4>"
-			var/logged_in = "/tg/ Station 13 Cargo Department"
+			var/logged_in = "[station_name] Cargo Department"
```
Comment (Cruix) [inline code/game/objects/items/devices/PDA/cart.dm]: This should be station_name(), not station_name. The proc will generate the name if it is currently blank.

File: code/modules/stock_market/computer.dm [hunk 2]
```diff
@@ -54,7 +54,7 @@ a.updated {
 </style>"}
 	var/dat = "<html><head><title>[station_name()] Stock Exchange</title>[css]</head><body>"
 
-	dat += "<span class='user'>Welcome, <b>[logged_in]</b></span><br><span class='balance'><b>Credits:</b> [balance()] </span><br>"
+	dat += "<span class='user'>Welcome, <b>[station_name] Cargo Department</b></span><br><span class='balance'><b>Credits:</b> [balance()] </span><br>"
```
Comment (Cruix) [inline code/modules/stock_market/computer.dm]: Same here

**Review by Cruix — CHANGES_REQUESTED** [review]

**Review by Cruix — APPROVED** [review]

**Comment by ShadowDeath6** [comment]
good pr

**Pull request merged**